test/lib/deployment/now-deploy.js

```diff
@@ -145,9 +146,10 @@ async function fetchWithAuth(url, opts = {}) {
     currentCount += 1;
     if (!token || currentCount === MAX_COUNT) {
       currentCount = 0;
-      if (process.env.NOW_TOKEN) {
+      if (process.env.VERCEL_TOKEN || process.env.NOW_TOKEN) {
         // used for health checks
         token = process.env.NOW_TOKEN;
+        token = process.env.VERCEL_TOKEN;
       } else {
```

Comment:
This doesn't seem right…

```suggestion
      // used for health checks
      token = process.env.VERCEL_TOKEN || process.env.NOW_TOKEN;
      if (!token) {
```